pages/docs/concepts/asyncapi-document/adding-bindings.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Adding Bindings
+weight: 260
+---
+
+Bindings in AsyncAPI provide a way to add protocol-specific information to the AsyncAPI documentation. They can be added to different document parts, such as servers, channels, or messages, to specify standard details specific to a particular protocol. The purpose of bindings is to enhance the understanding and usage of the API by providing additional context and configuration options for different protocols.
+
+```mermaid
+graph TD
+A[AsyncAPI Document] --> B((Servers))
+A --> D((Channels))
+D --> E((Messages))
+B --> C{Server Bindings}
+D --> F{Channel Bindings}
+E --> G{Message Bindings}
+```
+
+## Server Bindings
```

Comment:
We observe sentence capitalization for headers in our docs; please update all headers accordingly. 😄